cypress/e2e/tests/upgrade/create_upgrade_data.test.ts

```diff
@@ -110,6 +113,20 @@ describe(["@pre-upgrade"], "Creating pre-requisites before an upgrade", () => {
         tag.create();
     });
 
+    it("Creating and assess archetype", function () {
+        const { tagName, archetypeName } = this.upgradeData;
+        archetype = new Archetype(
+            archetypeName,
+            ["EJB XML", "Servlet"],
+            [tagName],
+            null,
+            [stakeHolder],
+            [stakeHolderGroup]
+        );
+        archetype.create();
+        archetype.perform_assessment("low", [stakeHolder], [stakeHolderGroup]);
```

Comment:
Include this statement to verify the completion of archetype assessment.
archetype.validateAssessmentField("Low");

Reply:
@ibragins I know there is a bug for validating status of archetype but it is targeted for 7.0.3 . SO let us validate status and it will be fixed soon.

Reply:
Done.